src/cladetime/util/session.py

```diff
@@ -25,7 +25,7 @@ def _get_session(retry: bool = True) -> requests.Session:
             total=5,
             allowed_methods=frozenset(["GET", "POST"]),
             backoff_factor=1,
-            status_forcelist=[401, 403, 404, 429, 500, 502, 503, 504],
+            status_forcelist=[429, 500, 502, 503, 504],
```

Comment:
There's no reason to retry 401, 403, and 404 status codes since we don't expect the result to change.